creator/jobs/models.py

```diff
@@ -69,7 +69,12 @@ def enqueued_at(self):
         ts = scheduler.connection.zscore(
             "rq:scheduler:scheduled_jobs", self.name
         )
-        dt = datetime.fromtimestamp(ts)
+        try:
+            dt = datetime.fromtimestamp(ts)
+        except TypeError:
+            # Sometimes this is returned as none from redis, perhaps if it is
```

Comment:
So what conditions cause redis to return None, do we know?

Reply:
Redis returns `None` when the key does no exist: https://redis.io/commands/zscore
I need to look into the internals to understand how the scheduler works, but it's also possible that the key is not there because we make the request during the time period between the execution of the task and the scheduling of the next event by the scheduler.

Reply:
So is the purpose of this `enqueued_at` function to get _any_ job's time of insertion into the queue or just scheduled jobs? If its the former, than have you tried querying redis for the job and looking at the enqueued_at property? `Job.fetch('my_job_id', connection=redis).enqueued_at`

Reply:
The purpose of this property is to get this specific Job's next scheduled queue time. The naming could be better because this is not the same as the `rq.Job().enqueued_at` property. `rq.Job().enqueued_at` specifies the time at which an `rq.Job` was enqueued for run on the _job_ queue. This `enqueued_at` property returns the time on the _scheduler_ queue at which a new `rq.Job()` will be enqueued on the _job_ queue.

Reply:
Ohh I see. Well other than the name of the property this PR looks ok? If you're considering the name maybe: `scheduled_at`

Reply:
Yeah we could change it to maybe `scheduled_for` or something but not likely in this PR as we have to make changes in the front end, too.